app/scripts/controllers/courtSearchInfo.ctrl.js

```diff
@@ -3,6 +3,7 @@
 angular.module('yourStlCourts').controller('CourtSearchInfoCtrl', function ($state, $window, court){
```

Comment:
Please revert everything in this file. None of these changes are needed. The variable isn't used and the semi-colon while not totally mandatory, keeps the code stricter.